easy_profiler_converter/converter.cpp

```diff
@@ -69,7 +69,7 @@ void JsonExporter::convertChildren(const profiler::reader::BlocksTreeNode& node,
         convert(child, children.back());
     }
 
-    json["children"] = children;
+    json["children"] = std::move(children);
```

Comment:
What's stopping the move assignment from happening already? `children` is an xvalue, yes?